packages/nx-plugin/.eslintrc.json

```diff
@@ -23,8 +23,7 @@
           "error",
           "@angular-devkit/architect",
           "@angular-devkit/core",
-          "@angular-devkit/schematics",
-          "@nrwl/workspace"
```

Comment:
We are using the workspace generator generator for tests, as well as the move generator